types/indy-sdk/index.d.ts

```diff
@@ -708,4 +709,11 @@ export interface GetNymResponse {
     role: NymRole;
 }
 
+export interface DidWithMeta {
+    did: Did;
+    verkey: Verkey;
+    metadata: string;
+    tempVerkey: Verkey;
```

Comment:
I don't see `tempVerkey` mentioned in the docs? 

https://github.com/hyperledger/indy-sdk/tree/main/wrappers/nodejs#listmydidswithmeta--wh----dids

Is it always returned?

Reply:
I took the info from the indy-sdk rust code (the nodejs code is a wrapper around this implementation):
https://github.com/hyperledger/indy-sdk/blob/1c7096dd95d0fd53881070f66907df4b9e61b874/libindy/src/commands/did.rs#L355

It is also in the nodejs test cases:  
https://github.com/hyperledger/indy-sdk/blob/1c7096dd95d0fd53881070f66907df4b9e61b874/wrappers/nodejs/test/did.js#L17

Reply:
But it does seem that both  `metdata` and `tempVerkey` are optional, https://github.com/hyperledger/indy-sdk/blob/1c7096dd95d0fd53881070f66907df4b9e61b874/libindy/src/domain/crypto/did.rs#L161-L166

Which for the Node.JS wrapper means they can be null I think?

Reply:
Good find! When i compare
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/bbff4b3696828af59cf6ae9132c47b4774df702d/types/indy-sdk/index.d.ts#L485
and

https://github.com/hyperledger/indy-sdk/blob/1c7096dd95d0fd53881070f66907df4b9e61b874/libindy/src/domain/anoncreds/credential.rs#L38

I updated my pr accordingly.